gradle-baseline-java/src/test/resources/com/palantir/baseline/palantirjavaformat-expected/BlankLines.java

```diff
@@ -4,9 +4,10 @@
     /**
      * Docstring that looks like a list:
      *
-     * <p>1. hey 2. there
+     * 1. hey
```

Comment:
why did this change?

Reply:
We bumped the version of palantir-java-formatter that we use in tests from 0.3.9 to 1.1.0 which added some changes to how the formatted output looks: 
https://github.com/palantir/gradle-baseline/pull/1534/files#diff-c2c619cc6bd6031ee10ada015c7ba9b841732e59d0a05d2a7408ebb0c02501e9L37-R37